test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -22,7 +22,7 @@ for i in "$@"; do
     esac
 done
 
-: "${NUM_WORKERS:=1}"
+: "${NUM_WORKERS:=0}"
```

Comment:
same as above: you will need to undo this change before submitting the upstream PR